README.md

```diff
@@ -1 +1,56 @@
-# Template3-project.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Template3 project.
+
+> This is project for template3 mobile version skeleton.
```

Comment:
- _(optional)_ Please give a descriptive project title to your readme file and also a short description of the project. The readme file is the first thing a recruiter looked when opening your project so its better to keep it very professional :)